### PR TITLE
GAA Regwall

### PR DESCRIPTION
**Pull request by ChrisAntaki**
This PR copies the metering demo from `swg-js`

### CODE REVIEW

File: app/views/gsi-iframe.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <!--
+      Publications should load this next script from:
+      https://news.google.com/swg/js/v1/swg-gaa.js
+    -->
+  <script src="<% swgGaaJsUrl %>"></script>
+
+  <script>
+    GaaGoogleSignInButton.show({
+      // Allow specific origins to get results from your Google Sign-In button.
+      allowedOrigins: ['http://localhost:8000', 'http://localhost:8080'],
```
Comment (elijahsoria) [inline app/views/gsi-iframe.html:11]: Should this include news.google.com i.e. SwG's production URL?

File: app/views/gsi-iframe.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+      // Specify your Google Sign-In Client ID.
```
Comment (elijahsoria) [inline app/views/gsi-iframe.html:13]: nit: Please add a comment as to how to get this ID (https://developers.google.com/identity/one-tap/web/guides/get-google-api-clientid or https://developers.google.com/identity/sign-in/web/sign-in#create_authorization_credentials)

File: app/views/gsi-iframe.html (hunk 1 above)
Comment (ChrisAntaki) [reply app/views/gsi-iframe.html]: I just added "https://scenic-2017.appspot.com", nice catch!

File: app/views/gsi-iframe.html (hunk 2 above)
Comment (ChrisAntaki) [reply app/views/gsi-iframe.html:17]: Added!

File: app/views/gsi-iframe.html (hunk 1 above)
Comment (elijahsoria) [reply app/views/gsi-iframe.html]: Great. Last nit would be to add a comment about why these origins are needed in the SwG context but that is up to you.

**Review by elijahsoria — APPROVED** [review]

File: app/views/gsi-iframe.html (hunk 1 above)
Comment (ChrisAntaki) [reply app/views/gsi-iframe.html]: I added comments for each allowed origin

**Pull request merged**